qcodes/instrument_drivers/QDevil/QDevil_QDac_channels.py

```diff
@@ -0,0 +1,833 @@
+# QCoDeS driver for the QDevil QDAC using channels
+# Adapted by QDevil from "qdev\QDac_channels.py" in the instrument drivers package - kudos to QDEV and the Microsoft quantum computing consortium for making that
+# AK QDevil 2019-06-24
+
+import time
+import visa
+import logging
+
+from datetime import datetime
+from functools import partial
+from operator import xor
+from collections import OrderedDict
+
+from qcodes.instrument.channel import InstrumentChannel, ChannelList
+from qcodes.instrument.channel import MultiChannelInstrumentParameter
+from qcodes.instrument.visa import VisaInstrument
+from qcodes.utils import validators as vals
+
+log = logging.getLogger(__name__)
+
+class QDacChannel(InstrumentChannel):
+    """
+    A single output channel of the QDac.
+
+    Exposes chan.v, chan.vrange, chan.vmin, chan.vmax, chan.slope, chan.i, chan.irange, chan.slope
+    Always set v to zero before changing vrange or irange
+    """
+
+    _CHANNEL_VALIDATION = vals.Numbers(1, 48)
+
+    def __init__(self, parent, name, channum):
+        """
+        Args:
+            parent (Instrument): The instrument to which the channel is
+                attached.
+            name (str): The name of the channel
+            channum (int): The number of the channel in question (1-48)
+        """
+        super().__init__(parent, name)
+
+        # Validate the channel
+        self._CHANNEL_VALIDATION.validate(channum)
+
+        # Add the parameters
+
+        self.add_parameter('v',
+                           label='Channel {} voltage'.format(channum),
+                           unit='V',
+                           set_cmd=partial(self._parent._set_voltage, channum),
+                           get_cmd='set {}'.format(channum),
```

Comment:
Well, it just so that several commands in this instrument are setters when called with more than one argument and getters when called only with one argument. But good point otherwise.

Reply:
So `set 0` is the visa command to get the voltage of channel 0? You should consider making the firmware accept `get`, too ;-).

Reply:
Yes, however "get" is already used for something else - unfortunately :-/